<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>HackTrip </string>
    </property>
    <widget class="QSpinBox" name="channelsSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="FPPcomboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FPP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>channels</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serverComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>jackloop128.stanford.edu</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>server</string>
     </property>
    </widget>
    <widget class="QComboBox" name="srateComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>48000</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>srate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>local</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="localUDPspinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>4464</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGUI_demo">
    <property name="title">
     <string>GUI demo</string>
    </property>
   </widget>
   <addaction name="menuGUI_demo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
